subjects/0-shell/audit/README.md

```diff
@@ -0,0 +1,85 @@
+#### Functional
+
+##### Open the project folder and confirm if the project was written in a compiled programming language.
+
+###### Can you confirm that the project was written in a compiled programming language?
+
+##### Open a terminal and run the project.
+
+###### Can you confirm that the project runs and displays a unix interpreter?
+
+##### Open a terminal and run the project.
+
+###### Can you confirm that this interpreter displays at least a simple `$` and waits for you to type a command?
+
+##### Try to open the project and run a command at your choice.
+
+###### Can you confirm that the interpreter only validates the command if you type enter?
+
+##### Try to open the project and run the command `exit`.
+
+###### Can you confirm that the interpreter terminates properly and gives back the parent's shell?
+
+##### Try to open the project and run the command `echo "something!"`. Do the same in your computer terminal.
+
+###### Can you confirm that the displayed message of the project is exactly the same as the computer terminal?
+
+##### Try to open the project and run the command `echo something else` (without double quotes). Do the same in your computer terminal.
+
+###### Can you confirm that the displayed message of the project is exactly the same as the computer terminal?
+
+##### Try to open the project and run the command `pwd`.
+
+###### Can you confirm that the interpreter displayed the current path?
```

Comment:
maybe you can add some questions like
` ##### Try to open the project and create a parent folder with two child folders using the command "mkdir". Then enter the parent folder and do "pwd" . `

`###### Can you confirm that the interpreter displayed the current path?`